detect.go

```diff
@@ -0,0 +1,45 @@
+package vscodiumbuildpack
+
+import (
+	"github.com/paketo-buildpacks/packit/v2"
+)
+
+type VersionParser interface {
+	ResolveVersion(cnbPath, version string) (resultVersion string, err error)
+}
```

Comment:
question: Why do we need this interface? Why not just directly put the `VersionParser` implementation in the `Detect` function?

Reply:
tests use faked implementations of these things: https://github.com/SwissDataScienceCenter/vscodium-buildpack/pull/2/files#diff-1b8e1524f4b6833ea54e94dac80da66b6add6aa6bf1037d39debd253ff26b419R5